R/exclude.R

```diff
@@ -55,6 +55,17 @@ is_excluded_file <- function(file_exclusion) {
   Inf %in% file_exclusion[[names2(file_exclusion) == ""]]
 }
 
+line_info <- function(line_numbers) {
+  n <- length(line_numbers)
+  if (identical(n, 0L)) {
+    "no lines"
+  } else if (identical(n, 1L)) {
+    paste("line", line_numbers)
```

Comment:
may as well use ngettext() here too since it's used below already

Reply:
could do instead of this one comparison with 1 - since that function treats 0 and 2 and above the same, I still need the 0 comparison above. So not sure what looks nicer. ok to keep the 0 and 1 comparisons?..

Reply:
I think the 0 case warrants its own handling here as much as I hate the extra branching